lib/matplotlib/pyplot.py

```diff
@@ -108,8 +109,14 @@ def _copy_docstring_and_deprecators(method, func=None):
 
 ## Global ##
 
-_IP_REGISTERED = None
-_INSTALL_FIG_OBSERVER = False
+
+_ReplDisplayHook = Enum("_ReplDisplayHook", ["NONE", "PLAIN", "IPYTHON"])
```

Comment:
Could you add a comment what this variable and the states mean?

Reply:
I guess I could just rename the variable to _CURRENT_INSTALLED_REPL_DISPLAYHOOK (or some variant thereof) and not put a comment?  Because the comment would likerally be "Whether the currently installed repl displayhook is no hook, a hook for plain python shells, or a hook for ipython shells", which seems quite redundant with the longer variable name?

Reply:
`_ActiveInputHook`?

Reply:
I've always been struggling with "repl_displayhook", but probably because 1) I wasn't aware of the concept behind it and 2) it's quite jargony. I think `install_repl_displayhook()` is not quite a good name, but sinc this here is private, I think
```suggestion
# The state controlled by [un]install_repl_displayhook()
_ReplDisplayHook = Enum("_ReplDisplayHook", ["NONE", "PLAIN", "IPYTHON"])
```
would be sufficient. Better explanation should be provided in the function docstring, which I've attepted in #22911.

Reply:
done